pkgs/by-name/da/daggerfall-unity/package.nix

```diff
@@ -0,0 +1,110 @@
+{
+  alsa-lib,
+  autoPatchelfHook,
+  copyDesktopItems,
+  fetchzip,
+  lib,
+  libGL,
+  libXScrnSaver,
+  libXcursor,
+  libXi,
+  libXinerama,
+  libXrandr,
+  libXxf86vm,
+  libpulseaudio,
+  libudev0-shim,
+  makeDesktopItem,
+  nix-update-script,
+  stdenv,
+  vulkan-loader,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "daggerfall-unity";
+  version = "1.1.1";
+
+  src = fetchzip {
+    url = "https://github.com/Interkarma/daggerfall-unity/releases/download/v${finalAttrs.version}/dfu_linux_64bit-v${finalAttrs.version}.zip";
+    stripRoot = false;
+    hash = "sha256-JuhhVLpREM9e9UtlDttvFUhHWpH7Sh79OEo1OM4ggKA=";
+  };
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  buildInputs = [
+    alsa-lib
+    copyDesktopItems
+    libGL
+    libXScrnSaver
+    libXcursor
+    libXi
+    libXinerama
+    libXrandr
+    libXxf86vm
+    libpulseaudio
+    libudev0-shim
+    vulkan-loader
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir --parents "$out"
+    cp --recursive * "$out"
```

Comment:
```suggestion
    mkdir -p "$out"
    cp -r * "$out"
```

Reply:
Does NixOS not come with GNU tools by default?

Reply:
Those are the shorthands everyone uses all the time and everyone knows and IMO I first needed to think a lot what the longhands actually mean because they are uses so rarely.

Reply:
I disagree. The long options are more beginner-friendly, because they are self-documenting.